src/quark.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use ffi;
+use translate::*;
+
+#[repr(C)]
+pub struct Quark(ffi::GQuark);
+
+impl Quark {
+    pub fn from_string(s: &str) -> Quark {
+        unsafe {
+            from_glib(ffi::g_quark_from_string(s.to_glib_none().0))
+        }
+    }
+
+    pub fn from_static_string(s: &'static str) -> Quark {
+        unsafe {
+            from_glib(ffi::g_quark_from_static_string(s.to_glib_none().0))
```

Comment:
This is wrong. The string passed to C must have static lifetime but due to conversion it doesn't here, is only in this scope.

We can't bind this function in a meaningful way due to the difference between C and Rust strings. The trailing 0

Reply:
So I just remove it as well?

Reply:
Yes

Reply:
Ok!